l10n_ch_invoice_with_payment/README.rst

```diff
@@ -0,0 +1,21 @@
+**This file is going to be generated by oca-gen-addon-readme.**
```

Comment:
This content is not the expected one :P 
See @Tonow-c2c 's comment about README generating.